sklearn/covariance/empirical_covariance_.py

```diff
@@ -104,13 +104,34 @@ class EmpiricalCovariance(BaseEstimator):
 
     Attributes
     ----------
+    location_ : array-like, shape (n_features,)
+        Estimated location, i.e. the estimated mean.
+
```

Comment:
Thanks, `ShrunkCovariance` is also missing it..

Reply:
Sorry, I thought I checked them all. Added now.